shell/browser/ui/win/notify_icon.cc

```diff
@@ -138,6 +138,16 @@ void NotifyIcon::DisplayBalloon(HICON icon,
     LOG(WARNING) << "Unable to create status tray balloon.";
 }
 
+void NotifyIcon::RemoveBalloon() {
+  NOTIFYICONDATA icon_data;
+  InitIconData(&icon_data);
+  icon_data.uFlags |= NIF_INFO;
+
+  BOOL result = Shell_NotifyIcon(NIM_MODIFY, &icon_data);
+  if (!result)
+    LOG(WARNING) << "Unable to remove status tray balloon.";
```

Comment:
As this stands, this will fail silently for end users. Do we think that's the best behavior in this case? We may want to throw an error or at least surface one through some means?

Reply:
`tray.displayBalloon()` has the same issue. If we want to propagate the error to the caller, it should be done in both APIs IMO

Reply:
i'd be in favor of that personally, what do you think?